dev/io.openliberty.io.opentelemetry.internal/bnd.bnd

```diff
@@ -13,6 +13,8 @@
 bVersion=1.0
 openTelemetryVersion=1.19.0
 
+src: src
```

Comment:
In a follow on PR, I think this bundle is now just a rebundling of io.opentelemetry and not an internal integration. Therefore (being a bit picky) the name changes to just `io.openliberty.io.opentelemetry`.